.travis.yml

```diff
@@ -17,44 +20,62 @@ cache:
     - $HOME/.composer/cache
     - $HOME/.config/composer/cache
 
+# Git clone depth.
+git:
+  depth: 1
+
 matrix:
   include:
-    - php: 5.4
-      env: WP_VERSION=4.9.10 PHP_LINT=1 ES_VERSION=1.3.0
```

Comment:
I'm not opposed to dropping support for PHP 5.4 and ES 1.3, but the README should be updated to reflect that

Reply:
Yep, I will make sure that I add that as part of this PR when it gets closer to completion. I grabbed the latest ES versions for each of the major branches (1, 2, 5, 6, 7) - is there something special about 1.3 versus 1.7? Did they change their versioning system at some point where 1.3.x is treated more as a major version?

Reply:
I honestly don't remember why that was. I _think_ that the significant changes weren't from 1.3.x -> 1.4.x, but rather from <1.3 -> 1.3, so I was testing 1.3 as the lowest-limit version. Then 1.7.x as the highest 1.x (1.3 and 1.7 probably are significantly different, but I couldn't tell you what impact those differences has on the SearchPress codebase).